game/src/layer/map.rs

```diff
@@ -30,7 +31,7 @@ impl Layer for BikeActivity {
         }
 
         // Show a tooltip with count, only when unzoomed
-        if ctx.canvas.cam_zoom < app.opts.min_zoom_for_detail {
```

Comment:
Lots of these traffic sim layers could be expressed as `Worlds`, kind of subsuming the `ColorNetwork` / `ColorDiscrete` helpers we've got now. I want to get to that eventually!